velox/docs/functions/spark/conversion.rst

```diff
@@ -2,6 +2,34 @@
 Conversion Functions
 ====================
 
+.. spark:function:: cast(value AS type) -> type
+
+    Explicitly cast a ``value`` to a specified ``type``.
+    Behavior when ANSI mode is disabled:
```

Comment:
> Behavior when ANSI mode is disabled:

Comment suggestion:

Follows the behavior when Spark ANSI mode is disabled, and does not support the behavior when ANSI is turned on.